src/util/bucket_logs_utils.js

```diff
@@ -43,16 +43,17 @@ async function export_logs_to_target(fs_context, s3_connection, bucket_to_owner_
  * This function gets a persistent log file, will go over it's entries one by one,
  * and will upload the entry to the target_bucket using the provided s3 connection
  * in order to know which user to use to upload to each bucket we will need to provide bucket_to_owner_keys_func
- * @param {nb.NativeFSContext} fs_context
+ * @param {Object} config_fs
```

Comment:
you can set the type with inline import like [this](https://github.com/noobaa/noobaa-core/blob/55d87ae07139c162f1870fff0732af678778d6fc/src/endpoint/s3/s3_utils.js#L676), or import it once with a typedef like [this](https://github.com/noobaa/noobaa-core/blob/55d87ae07139c162f1870fff0732af678778d6fc/src/util/native_fs_utils.js#L20).
```suggestion
 * @param {import('../sdk/config_fs').ConfigFS} config_fs
```